solvers.tex

```diff
@@ -60,7 +60,7 @@
 >>> solveset(x - x, x, domain=S.Reals)
 (-oo, oo)
 >>> solveset(x - x, x, domain=S.Complexes)
-S.Complexes
```

Comment:
I see. It's used for hash stability. Let's just skip this test (and the mpmath one above).